libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.ts

```diff
@@ -573,6 +573,10 @@ export class SkyTextEditorComponent
       this.#adapterService.focusEditor();
     }
 
+    if (this.disabled) {
+      this.disabled = true;
```

Comment:
I'm confused by this line? If `true`, set to `true`?

Reply:
Re-run the setter after the iframe is initialized.

Reply:
This could be made clearer by moving lines 107-132 into a separate `#updateDisabledState` method and calling it both in the setter, and here. Otherwise, I imagine a future encounter with this block of code will create the same level of confusion.

Reply:
Good call. Updated.